client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -333,8 +462,8 @@ class CancelPurchaseForm extends React.Component {
 			.filter( ( product ) => product.term === TERM_ANNUALLY )
 			.map( ( product ) => ( {
 				value: product.productSlug,
-				label: translate( 'Jetpack {{planName/}}', {
-					components: { planName: <>{ product.shortName }</> },
+				label: translate( 'Jetpack %(planName)s', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.